.env.example

```diff
@@ -1 +1,2 @@
+API_URL=
```

Comment:
I added this since there is currently fetching being done on the server, compared to previously where it was done both server-side and client-side